README.md

```diff
@@ -18,11 +18,15 @@ Install:
 npm install
 ````
 
+#### Initialize the database
+
 Put your datafiles in `data`, then do:
 
 ```
-docker-compose -f docker-compose-init-db.yml rm -f
-docker-compose -f docker-compose-init-db.yml up
+npm run serve-dev # or npm run serve-prod in production environment
+
+# then in another window
+docker exec 6element_backup_1 tools/init-database.js
 docker-compose -f docker-compose-load-db.yml up
 ```
```

Comment:
why can't we do the same way as initdb `docker exec 6element_backup_1 tools/load-database.js`